src/images/pages/regulation/map.svg

```diff
@@ -1,3565 +1 @@
-		l0.6,0.6v0.6h1.1v1.2h1.1v0.6l0.6,1.1h1.7v1.8l-0.6,0.6l0.5,0.6h1.1v-0.6h0.7v0.6l0.5,0.5v0.6l0.6,0.5v0.6h0.5l0.6,0.6l0.6,0.6v0.4
-		l0.5-0.5h1.3l0.6,0.5h1.1l1.1,2.3l0.6,0.5l1.1,0.6h1.1l0.5,0.6h1.1l0.6,0.6l-1.1,1.1l-0.6,0.6l-0.5,1.1v0.6l-0.6,0.6v2.2l-0.5,0.6
-		v1.8l-0.6-0.6l-0.5,1.1l-0.7-0.6l-0.5-0.5l-0.5,0.5v0.6l-0.6,1.1l-1.1,0.6h-1.1l-0.5,0.5v0.6l-1.2,1.1l-0.5,1.1l0.5,1.1l1.1-0.5
-		l0.5-1.2h0.6v-0.6h0.6l0.6,1.1v0.7h-1.1v0.5l0.6,0.6h-0.6v0.5l0.5,0.5v1.2h-0.5v0.5l0.5,0.6v0.6l0.6,0.6l-0.6,0.6l-0.5,0.6
-		l-0.6,0.6h-1.7v0.5l0.5,0.5h0.6l0.6,0.6v1.1l-0.5,0.5v2.3l0.5,1.1h1.7l0.6,0.6l-0.6,0.6v0.6l-1.1,1.1l0,0l-0.7,0.7v-0.6h-1.1v0.6
-		h-1.6v1.1l-0.6,0.6h-2.9l-0.6-0.6v-0.6l-1.1-1.1l-0.6,0.6v-1.2h-1.1v-1.1h-0.5l-0.6,0.5h-0.6l-0.6,0.6v-1.2h-1.7l-0.6-0.6l-0.5-0.6
-		h-0.6l-1.2,1.1h-1.7l-0.6,0.6H121l-0.5,0.5v0.5l-0.6,1.1l0.6,0.6l-0.6,0.6l0.1,0.3l0.2,0.3l0.3,0.2L120,160.8z M119.5,160.4
-		L119.5,160.4l0.4,0.3l0.6-0.5l-0.2-0.2l-0.2-0.3l-0.1-0.3l0.6-0.6l-0.6-0.6l0,0l0.6-1.1v-0.6l0.6-0.6h0.5l0.6-0.6h1.7l1.2-1.1h0.7
-		l0.6,0.6l0.6,0.6h1.7v1.1l0.5-0.5h0.6l0.6-0.6h0.6v1.1h1.1v1l0.5-0.5l1.2,1.2v0.6l0.5,0.5h2.8l0.5-0.5v-1.2h1.7V157h1.2v0.5l1.1-1
-		l0.6-0.6v-0.6l0.5-0.6l-0.5-0.5h-1.7l-0.6-1.1v-2.3l0.5-0.5v-1.1l-0.5-0.6h-0.6l-0.6-0.6v-0.6h1.8l0.6-0.6l0.5-0.6l0.5-0.5
-		l-0.6-0.5v-0.6l-0.5-0.6v-0.6h0.5v-1.1l-0.5-0.5v-0.6h0.5l-0.5-0.5v-0.6h1.1v-0.6l-0.6-1.1h-0.4v0.6h-0.6l-0.5,1.1l0,0l-1.2,0.6
-		l-0.6-1.2l0.6-1.1l1.2-1.1v-0.6l0.6-0.6h1.1l1.1-0.6l0.6-1.1v-0.6l0.6-0.6l0.6,0.6l0.6,0.5l0.5-1.1l0.5,0.5v-1.6l0.5-0.6v-2.2
-		l0.6-0.6v-0.5l0.6-1.1l0.6-0.6l1.1-1.1l-0.5-0.5h-1.1l-0.5-0.6h-1.1l-1.1-0.6l-0.6-0.6l-1.1-2.3h-1.1l-0.6-0.5h-1.1l-0.6,0.6v-0.6
-		l-0.6-0.6l-0.6-0.6h-0.6v-0.6l-0.6-0.5v-0.6l-0.5-0.5v-0.6h-0.5v0.6h-1.2l-0.6-0.7l0.6-0.6V114h-2l-0.6-1.2v-0.5H132v-1.2h-1.1
-		v-0.6l-0.6-0.6v-1.7H127l-0.6,0.6v1.1l-0.5,0.6v2.3h-1.2l-0.6,0.5h-0.6l-1.1,0.6l-1.2,0.5H119v0.5h0.6l1.1,0.6l0.7,0.7h-0.7v0.6
-		l-1.2-0.6h-2.3v-0.6h-1.1l-1.1-0.5l-1.1-0.6v0.6l-1.3-0.7l0.6-0.6l-0.6-0.6v-0.6l0.5-0.5h-1l-0.6-0.6H111v0.6l-0.6,0.6v1.6l0.6,1.2
-		l-0.6,0.6v1.1l0.6,0.6v0.6h-1.2l-0.6-0.5H108l0.5-0.6h-2.2l-0.5-0.4v0.4h-1.2v-1.7h-1.2V115l-1.1-0.6l-0.5,0.6v0.6h-1.2v-0.5h-1.7
-		l-0.5-0.6l-0.6,0.6h-1.6v0.4h1.6v1.3h-0.5v0.5h-1.2l-0.5,0.5h0.5l0.6,0.6l0.5,1V119l1.1,0.6v-0.5h0.6v0.6l1.1,1.1h0.6v0.6h0.5v1.1
-		h2.4l-0.6,0.6h0.5l0.6,0.6l-0.5,1.1h0.5l0.6,0.6h1.1v0.1h-1.1v0.5h0.5v0.6l-1.1,1.2v0.5l0.6,1.1h0.5v1.2l0.6,0.6v0.5l1.1,0.5
-		l0.6-0.6v1.3l-0.5,0.5h0.5l-0.6,0.6v1.7l-0.6,0.6v0.5h0.6v1.1l0.6,0.6v2.4l-0.7-0.6V139l-0.6-0.6v-0.6l-0.6-0.5v-0.5l-0.5,0.5v2.2
-		l-0.5,0.6v1.1H104l1,0.5l-0.6,0.6h-0.7v1.1l-0.6,0.6v0.6h-0.5v1.1H102l-0.6,0.6v0.6l0,0l-1.1,1.1l-1.1,1.1l0.6,0.5l0.6,0.6v1.1h1.1
-		l1.2,1.1h0.5l0.6,1.1h1.1l0.5,0.5l1.2,1.1h0.6l1.1,0.6h0.6v0.6l1.3,0.1l1.3-0.4l0.7,0.5l0.2,0.7l0,0l-0.1,0.8l0.5,0.5h1.7l0.6,0.6
-		l0.6,0.6h1.7l0.6,0.5l0.6,0.1l0.5-0.1L119.5,160.4z"/>
-	<polygon class="st0" points="104.2,82.2 103.7,82.7 103.7,82.2 103,82.2 103.
```

Comment:
there's a reason we didn't flatten this file, I believe this will now break our links on the svg. please check if it does and add an exception for this file to keep it unoptimized.

Reply:
So, as discussed, it might not be necessary to add an exception for this.

Reply:
as checked together seems it's not needed to keep it anymore and everything works fine. @binary-hamedanchi do you remember why we kept the unoptimized version of this file? I think it was for the map of coordinates that we had before, but as we checked now everything seems to be done dynamically.